index.html

```diff
@@ -0,0 +1,388 @@
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        
+      </div>
+      <div class="about-me">
+        <section class="about-me-section">
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+
+    <!-- Contact form start -->
+    <footer id="contact" class="footer">
+      <div class="form-container">
+        <p class="contact-msg">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in
+          touch.
+        </p>
+        <form action="https://formspree.io/f/" method="POST" class="form">
+          <div class="form-control">
+            <input id="name" name="user_name" placeholder="Full Name" />
```

Comment:
- [ ] It could be nice if you add into your input tag a `type` attribute because the possible entries and behavior of the `<input>` tag strongly depend on the value indicated in its `type` attributes. In this case, it will be of type `text`. You can read more about different HTML input types and their uses in this [article](https://www.w3schools.com/html/html_form_input_types.asp).
- [ ] You can also provide a `required` attribute to specify that the input field must be filled out before submitting the form.
- [ ]  As required, the Name input  has `maxlength="30"` attribute. Please insert it too.